src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +26,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super T> machines, Object value);
```

Comment:
you forget about method param. It should be generic as well because we dont want `to fill` the `String` object and etc. 
```suggestion
    void fill(List<? super T> machines, T value);
```